core/scss/utilities/functions/modular-scale/_get-modular-spacing-base.scss

```diff
@@ -0,0 +1,15 @@
+@charset "UTF-8";
+
+//
+// get-modular-spacing-base($breakpoint)
+//
+// Get a value out of the modular-spacing-base map
+// Returns the pixel value of the $modular-spacing-base variable at a breakpoint.
+//
+// $breakpoint - One of `xs`, `sm`, `md`, `lg`, `xl`, `2xl`
+//
+// Style guide: Functions.ModularScale.get-modular-spacing-base
+//
+@function get-modular-spacing-base($breakpoint: 'xs') {
+  @return map-get($modular-spacing-base, $breakpoint);
```

Comment:
map-get returns `null` if the key is missing. We should add a warning here if the breakpoint key goes missing. It is likely that someone will modify the grid and breakpoints in the future.